File: DAO/cancelacion.py
class Cancelacion():
    def __init__(self, codigo=0, fecha="", montoReintegro=0, registro = None):
        if(not(registro is None)):
            self._codigo = registro[0]
            self._fecha = registro[1]
            self._montoReintegro = registro[2]

        else:
            self._codigo = codigo
            self._fecha = fecha
            self._montoReintegro = montoReintegro

    @property
    def codigo(self):
        return self._codigo

    @codigo.setter
    def codigo(self, codigo):
        self._codigo = codigo

    @property
    def fecha(self):
        return self._fecha

    @fecha.setter
    def fecha(self, fecha):
        self._fecha = fecha

    @property
    def montoReintegro(self):
        return self._montoReintegro

    @montoReintegro.setter
    def montoReintegro(self, montoReintegro):
        self._montoReintegro = montoReintegro

    def datos(self):
        return (self._codigo,
                self._fecha,
                self._montoReintegro,)

    def __str__(self):
        return str(self.__dict__)


if __name__ == '__main__':
    pass

File: JAVA/cgiMiercoles/src/datos/Cancelacion.java
package datos;

import java.sql.Date;
import java.util.GregorianCalendar;

public class Cancelacion {
	private String pasajeCancelado;
	private String cOrigen;
	private String cDestino;
	private Date fecha;
	private float montoReintegro;
	public String getPasajeCancelado() {
		return pasajeCancelado;
	}
	public void setPasajeCancelado(String pasajeCancelado) {
		this.pasajeCancelado = pasajeCancelado;
	}
	public String getcOrigen() {
		return cOrigen;
	}
	public void setcOrigen(String cOrigen) {
		this.cOrigen = cOrigen;
	}
	public String getcDestino() {
		return cDestino;
	}
	public void setcDestino(String cDestino) {
		this.cDestino = cDestino;
	}
	public Date getFecha() {
		return fecha;
	}
	public void setFecha(Date fecha) {
		this.fecha = fecha;
	}
	public float getMontoReintegro() {
		return montoReintegro;
	}
	public void setMontoReintegro(float montoReintegro) {
		this.montoReintegro = montoReintegro;
	}
	public Cancelacion(String pasajeCancelado, String cOrigen, String cDestino, Date fecha, float montoReintegro) {
		super();
		this.pasajeCancelado = pasajeCancelado;
		this.cOrigen = cOrigen;
		this.cDestino = cDestino;
		this.fecha = fecha;
		this.montoReintegro = montoReintegro;
	}
	@Override
	public String toString() {
		return "Cancelacion [pasajeCancelado=" + pasajeCancelado + ", cOrigen=" + cOrigen + ", cDestino=" + cDestino
				+ ", fecha=" + fecha + ", montoReintegro=" + montoReintegro + "]";
	}

}
